assets/src/components/supervisor-block.tsx

```diff
@@ -84,7 +115,97 @@ export const SupervisorBlock = ({ item }: { item: ApiSupervisor }) => {
         content: (
           <div>
             {value.map((process, index) => {
-              return <ProcessRow process={process} server={item.server} key={index} />;
+              return <div className='flex h-10 flex-row' key={index}>
+                <div className='flex flex-grow items-center'>
+                  <div className='px-2'>
+                    <p>
+                      <a
+                        className='transititext-primary text-sm text-primary transition duration-150 ease-in-out hover:text-primary-600 focus:text-primary-600 active:text-primary-700 dark:text-primary-400 dark:hover:text-primary-500 dark:focus:text-primary-500 dark:active:text-primary-600'
+                        data-te-toggle='tooltip'
+                        title={process.name}
+                      >
+                        {trimForProcessBlock(process.name)}
+                      </a>
+                    </p>
+                  </div>
+                </div>
+                <div className='flex flex-nowrap items-center space-x-2'>
+                  {landingStore.isAllowMutatorsActive && (
+                    <>
+                      <ActButton onClick={() => landingStore.removeProcess(item.server, process)} type={Type.Red}>
+                        <GrErase/>
+                      </ActButton>
+                      <ActButton onClick={() => landingStore.cloneProcess(item.server, process)} type={Type.Blue}>
+                        <GoDuplicate/>
+                      </ActButton>
+                    </>
+                  )}
+                  {process.outLog && (
+                    <ProcessLog log={process.outLog} server={item.server} process={process}
+                                purpose={ProcessLogPurpose.StdOut}/>
+                  )}
+                  {process.errLog && (
+                    <ProcessLog log={process.errLog} server={item.server} process={process}
+                                purpose={ProcessLogPurpose.StdErr}/>
+                  )}
+                  <div className={'flex flex-col items-center'}>
+                    {('RUNNING' === process.stateName && (
+                        <span
+                          className={
+                            'inline-block whitespace-nowrap rounded-[0.27rem] bg-success-100 px-[0.65em] pb-[0.25em] pt-[0.35em] text-center align-baseline text-[0.75em] font-bold leading-none text-success-700'
+                          }
+                        >{process.stateName}</span>
+                      )) ||
```

Comment:
this is very crazy code, fix it 😱😱😱😱😱😱😱😱

Reply:
outdated